.gitignore

```diff
@@ -1,3 +1,21 @@
+/app/config/parameters.yml
+/build/
+/phpunit.xml
+/var/*
+!/var/cache
+/var/cache/*
+!var/cache/.gitkeep
+!/var/logs
+/var/logs/*
+!var/logs/.gitkeep
+!/var/sessions
```

Comment:
Sizin komitleri symfoni hakkında bilgi öğrenmek için kullanayım diyorum. İlk sorum şu, symfony session bilgisini dosyaya mı yazıyor? Bu satırdan bunu çıkarıyorum.